src/test/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreDocumentAdapterTest.java

```diff
@@ -97,9 +97,22 @@ public void createDocument() {
         given(collectionReference.document()).willReturn(documentReference);
         given(batch.create(any(DocumentReference.class), anyMap())).willReturn(batch);
 
-        adapter.createDocument(collectionPath, fields, batch);
+        DocumentReference ref = adapter.createDocument(collectionPath, fields, batch);
         verify(collectionReference).document();
         verify(batch).create(same(documentReference), anyMap());
+        assertEquals(documentReference, ref, "Should return the document reference created");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Should return the document reference created" appears 7 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5465948)